.github/actions/multi-functest/action.yml

```diff
@@ -143,7 +145,7 @@ runs:
           nix-verbose: ${{ inputs.nix-verbose }}
           gh_token: ${{ inputs.gh_token }}
           custom_shell: ${{ inputs.custom_shell }}
-          cflags: ${{ inputs.cflags }}
+          cflags: "${{ inputs.cflags }} -DMLD_FORCE_RISCV64"
```

Comment:
There is no `MLD_FORCE_RISCV64` flag yet. Add it?

Reply:
Done